packages/amplify-category-geo/src/service-walkthroughs/mapWalkthrough.ts

```diff
@@ -79,12 +80,12 @@ export const mapAdvancedWalkthrough = async (context: $TSContext, parameters: Pa
     // const includePricingPlan = await geoServiceExists(ServiceName.Map) || await geoServiceExists(ServiceName.PlaceIndex);
     const includePricingPlan = false;
     const currentPricingPlan = parameters.pricingPlan ? parameters.pricingPlan : await getGeoPricingPlan();
-    context.print.info('Available advanced settings:');
-    context.print.info('- Map style & Map data provider (default: Streets provided by Esri)');
+    printer.info('Available advanced settings:');
+    printer.info('- Map style & Map data provider (default: Streets provided by Esri)');
     if (includePricingPlan) {
-        context.print.info(`- Map pricing plan (current: ${currentPricingPlan})`);
+        printer.info(`- Map pricing plan (current: ${currentPricingPlan})`);
```

Comment:
nit: use `formatter.list()` to print a list of values

Reply:
Nice. Will use it